apps/website/gatsby-node.mjs

```diff
@@ -17,11 +17,20 @@ export const onCreateWebpackConfig = ({ actions }) => {
   });
 };
 
+/**
+ * @template T extends any
+ * @param {T | undefined | null} val
+ * @returns {val is T}
```

Comment:
Wow! Did not know it's possible :mindblown:

Reply:
Yeah, its awesome! Perfect compromise between no type checking and the catastrophy of typescript in gatsby-node.